runtime/services/multimediad/index.js

```diff
@@ -9,6 +9,9 @@ var Flora = require('./flora')
 var Dbus = require('dbus')
 var logger = require('logger')('multimediad')
 var Remote = require('../../lib/dbus-remote-call.js')
+var system = require('@yoda/system')
+
+system.adjustMallocSettings(13)
```

Comment:
@legendecas How about moving this into oh-my-little-pony for all services?

Reply:
Does it have any side effects?

Reply:
I'm not sure that, just wonder applying this on other services as the global config.

Reply:
Yet doesn't see the point to apply to other services. 🤔